src/Microsoft.ML/Runtime/Internal/Tools/CSharpGeneratorUtils.cs

```diff
@@ -279,10 +280,22 @@ private static string GetCharAsString(char value)
                 case TlcModule.DataKind.Bool:
                     return (bool)fieldValue ? "true" : "false";
                 case TlcModule.DataKind.Enum:
+                    string enumAsString = fieldValue.ToString();
```

Comment:
It is something of an unfortunate engineering situation that the so-called C# API code generator and the manifest JSON generator utilize, as near as I can see, entirely separate codepaths to interact with the same structures.